---
title: About
permalink: /about/
---

### About us
We are a highly collaborative and interactive team that studies the interaction of fungi and plants. We use a wide variety of tools reaching from biochemistry, bioinformatics, genomics, to environmental sampling. Modern biology is best mixed by combining good 'old school' biochemistry with modern data science stirred with interesting biological questions.

### How to join our Team
We welcome people from all ways of life, fostering diversity and inclusion. Please see [How to Join](http://Team-Schwessinger.github.io/Team_B_S/how_to_join/) for more information.

### Research

Our team's main focus revolves around questions and hypotheses related to (plant) biosecurity in Australia by studying the interaction between plants, their fungal pathogen, and their phyllosphere microbiomes. Specifically, we work on agronomic important wheat rust fungi ([**stripe**](https://en.wikipedia.org/wiki/Wheat_yellow_rust), [leaf](https://en.wikipedia.org/wiki/Wheat_leaf_rust), and [stem](https://en.wikipedia.org/wiki/Stem_rust)) and the environmental important [myrtle rust fungus](https://www.dpi.nsw.gov.au/biosecurity/plant/insect-pests-and-plant-diseases/myrtle-rust) that infects a wide range of myrtacea (e.g. eucalyptus trees). Concomitantly, we work on novel sequence based diagnostics combining nanopore long-read sequencing and machince learning.  

In addition, we work highly collaboratively to enable exciting science and public health efforts. Current joined projects are:

* on the genome biology of [*Wahlenbergia ceracea*](https://en.wikipedia.org/wiki/Wahlenbergia_ceracea) in collaboration with the [Nicotra Lab](https://biology.anu.edu.au/people/academics/adrienne-nicotra) 
* supporting the public health response to COVID-19 in the Australian Captial Territory (ACT) by performing SARS-COV2 genome sequencing in collaboration with ACT Pathology and ACT Health. 



**Key questions are:**

* How do invasive fungal pathogens infect host plants?
* What are the genes underlying host adaptation enabling infection?
* How does evolution and adaptation shape genome structure in agricultural vs. natural ecosystems?
* Can we develop cutting edge sequence based methods for pathogen detection?
* How do microbial communities in plant leaves change during the infection with invasive pathogens?

Many of these questions are also applicable to other study systems, hence we are a really collaborative team. We work with other groups at RSB and beyond on plant, animal, and fungal genomics. 

**Specifics**

We study plant-fungi interactions on multiple molecular and temporal scales. This includes many aspects:

* Detailed molecular interaction studies of plant and fungal proteins leading to disease or immunity. The seconds to days timescale.
* Detailed genome evolution and adaptation of fungal pathogens, especially the [wheat stripe rust fungus](https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.14159?scrollTo=references&), in response to wild and agricultural ecosystems. The months to years timescale.
* Detailed comparative and population genomics analysis of rust fungi with a focus on wheat rusts, yet looking beyond. The decades to millions of year timescale.

Our mid-2020/early-2021 lab projects include: 

[Fungal genomics, evolution, and host adaptation](https://biology.anu.edu.au/research/projects/fungal-genomics-evolution-and-host-adaptation) (Summer, Honours, Master, and PhD students)

[Detection of fungal pathogens and their associated microbiome](https://biology.anu.edu.au/research/projects/detection-fungal-pathogens-and-their-associated-microbiome) (Summer, Honours, and Master students)

**Cutting edge genome biology**

Evolution has driven genetic diversity of life on Earth, but also created highly complex genomes that are difficult to analyse. Current draft genomes can have thousands to hundreds of thousands of contigs rather than chromosomes, containing incorrect assemblies, gaps and errors. With rapid advances in long-read and long-range genomic technologies, it is becoming possible to resolve complex genomes, including repetitive, polyploid plant genomes. Despite theses technologies being available, a challenge persists: the extraction of pure high molecular weight DNA suitable for long-read sequencing. This is particularly true of recalcitrant native Australian trees such as Eucalypts and Acacias and many fungal species. To resolve this we are actively developing protocols for HMW DNA extraction from challenging tissues and share these protocols once optimsed on protocols.io [here](https://www.protocols.io/groups/high-molecular-weight-dna-extraction-from-all-kingdoms/publications) and [here](https://www.protocols.io/groups/team-schwessinger). Such ultra-long reads assist the assembly of high quality genomes and can generate complete chromosomes when cscaffolded with [HiC technology](https://phasegenomics.com/).

### The science beyond the 'lab'
Beyond being nerds, we aim to contribute to a welcoming and progressive academic environment. Group members are actively encouraged to contribute to community events, outreach science communication, and university commities. For example, group members have been part of [eLife's Early Career Advisory Board](https://elifesciences.org/), ambassador for the protocol sharing webpage [protocols.io](https://www.protocols.io/), and the chair of the RSB Equity Committee. And of course we love open access and reproducible science. For more information and detail on our contributions to open science, please look at our [Open Science](http://teamschwessinger.com/open_science/) page.

<hr>

### **Funding and Grants (>=$10, 000 AUD)**

| Granted to | When | What | Grant type | Amount (AUD) | Role |
| :----------- | :----------- | :----------- | :----------- | :----------- | :----------- |
| B. S. | 07/2020 - 06/2022 | Australian Capital Territory Priority Investment Program (PIP) - EcoKDDart: Future Proofing Biodiversity and Biosecurity with a novel integrative Big Data Analysis Hub | Industry Research Grant | $1.2 million | co-Principal Investigator |
| B. S. | 06/2020 - 05/2022 | Australian Research Council Linkage project - Combating myrtle rust, a new disease threatening Australiaâ€™s unique flora | Research Grant | $400k | Principal Investigator |
| B. S. | 02/2019 | Mozilla Science Foundation, Reproducibility for Everyone | Science Grant | $15k | Principal Investigator |
| B. S. | 11/2018 | Research School Biology, Increasing Scientific Rigor at RSB | Science Grant | $10k | Principal Investigator |
| B. S. | 10/2018-09/2022 | Australian Research Council - Identify, characterise, detect factors causing wheat disease epidemics | Research Grant | $780k | Principal Investigator |
| B. S. | 07/2017-06/2019 | Hermon Slade Foundation, Rapid detection and diagnosis of plant pathogens | Innovation Grant | $50k | co-Principal Investigator |
| B. S. | 05/2015-04/2017 | Bioplatform Australia, Sequencing grant for wheat stripe rust fungi | Sequencing Grant | $30k | co-Principal Investigator |
| B. S. | 09/2015-09/2018 | Australian Research Council, Understanding the genomic and epigenomic mechanisms driving the evolution of highly virulent stripe rust races | Research Grant | $350k | Principal Investigator |
| B. S. | 07/2012-09/2015 | Human Frontier Science Program Long-Term Fellowship, Function of non-RD kinases in plant immunity | Postdoctoral Grant | $195k | Principal Investigator |
| B. S. | 01/2012-07/2015 | European Molecular Biology Organization Long-Term Fellowship, Function of non-RD kinases in plant immunity | Postdoctoral Grant | $80k | Principal Investigator |



### **Awards, Fellowships and Scholarships**

| Awarded to | When | Award type | What | 
| :----------- | :----------- | :----------- | :----------- | 
| S. W. | 05/2022 | Scholarship | Ritman Scholarship for the PBRI Symposium 2022 |
| T. E. | 05/2022 | Scholarship | Ritman Scholarship for the PBRI Symposium 2022 |
| S. W. | 08/2021 | Scholarship | Crawford Fund Conference Scholarship 2021 | 
| B. S. | 10/2018-09/2022 | Fellowship | Future Fellowship 2018 - Australian Research Council |
| B. S. | 01/2017 | Award | Shortlisted for the New Phytologist Tansley Medal for Excellence in Plant Science |
| B. S. | 10/2015-09/2018 | Award | Discovery Early Career Researcher Award - Australian Research Council |
| B. S. | 07/2012-09/2015 | Fellowship | Human Frontier Science Program Long-Term Fellowship |
| B. S. | 01/2012-06/2012 | Fellowship | European Molecular Biology Organization Long-Term Fellowship |
| B. S. | 07/2011 | Award | John Innes Foundation Prize for Excellence in Scientific Research |
| B. S. | 10/2006-09/2010 | Scholarship | Rotation PhD Scholarship John Innes Centre and The Sainsbury Laboratory |
| B. S. | 09/2006 | Award | Short-listed for the SET Student of the Year Awards in 2006 |
| B. S. | 07/2006 | Scholarship | Plant Science Scholarship of the University of Glasgow |
| B. S. | 06/2005 | Scholarship | Lang Scholarship (Botany) of the University of Glasgow |
| B. S. | 05/2005-06/2006 | Scholarship | Sainsbury Undergraduate Studentship in Plant Science |
| B. S. | 10/2002-07/2006 | Scholarship | Scholarship for Intellectually Gifted People of the State of Bavaria |
| B. S. | 09/2002-09/2010 | Scholarship | Scholarship of e-fellows.net |
